src/lib/radio/radio.ts

```diff
@@ -239,6 +239,9 @@ export class MdRadioButton implements OnInit {
   @HostBinding('class.md-radio-focused')
   _isFocused: boolean;
 
+  /** Tabindex for the input element. */
+  @Input() tabindex: number = 0;
```

Comment:
The property should be `tabIndex`

I'm confused why this wasn't working, though- focus should have gone to the native radio without having to explicitly set a tabindex. Where were you seeing the problem?